webrtc.html

```diff
@@ -3312,8 +3322,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dd>
         </dl>
 
-        <dl class="idl" title="dictionary RTCRtpCodecCapability">
-          <dt>DOMString mimeType</dt>
+        <dl class="idl" title="interface RTCRtpCodecCapability">
+          <dt>readonly attribute DOMString mimeType</dt>
```

Comment:
If this is not an interface, the application can construct preferred codecs that the browser didn't agree to supporting.